features/all-positions/PositionActionsDialog.tsx

```diff
@@ -489,19 +504,69 @@ const PositionActionsDialog = (props: DialogProps) => {
                           price.
                           <br></br>
                           <br></br>
+                          <Status>
+                            <Label>Current collateral/token ratio: </Label>
+                            {collRatio.toFixed(4)}
+                          </Status>
+                          <Status>
+                            <Label>Minimum profitable ratio: </Label>
+                            {minCollPerTokenToBeProfitable}
+                          </Status>
+                          <Status>
+                            <Label>Maximum liquidatable ratio: </Label>
+                            {maxCollPerTokenToBeValid}
+                          </Status>
                         </Typography>
-                        <Important>
-                          Exercise caution! Incorrectly liquidating a position
-                          can lose you money! See the{" "}
-                          <a
-                            href={DOCS_MAP.FINAL_FEE}
-                            target="_blank"
-                            rel="noopener noreferrer"
-                          >
-                            UMA docs
-                          </a>
-                          .
-                        </Important>
+                        {Number(minCollPerToken) <
```

Comment:
There's a couple of `Number(minCollPerToken)` thoughout the place. Seems like its converting it from a string. Should we take into account that its a NaN?

Reply:
Great catch, I initialized new variables before the render like so:

```
const minCollPerTokenNum = Number(minCollPerToken) || 0;
```